services/servo.md

```diff
@@ -2,15 +2,18 @@
 
     identifier: 0x12fc9103
 
-Servo is a small motor directed with a PWM signal.
-This services fixes the servo period at 20ms, and the pulse can be regulated.
+Servo is a small motor with arm that can be pointing at a specific direction.
 
 ## Registers
 
-    rw pulse: u32 us {typical_min = 500, typical_max = 2500} @ value
+    rw angle: i16 ° {typical_min = -90, typical_max = 90} @ value
```

Comment:
I would be inclined to use `deg` as the unit name; I think we use ASCII names everywhere

Reply:
I think we can have more resolution than 1 deg (the previous one had 1us out of ~1000, so ~0.2deg). Maybe we should use `i16.16` type ? (both here and for drift)

Reply:
I'll address the ° issue in a later PR. It's currently used in the display which should be °.